tools/accuracy_checker/accuracy_checker/metrics/coco_orig_metrics.py

```diff
@@ -62,11 +62,18 @@ def parameters(cls):
             'threshold': BaseField(optional=True, default='.50:.05:.95', description='threshold for metric calculation')
         })
 
+        parameters.update({
+            'add_1_to_bbox_side': BoolField(
```

Comment:
for me `include_boundaries` sounds better

Reply:
I decided not to include it in this PR, removed